processor/configuration/src/main/resources/schemas/sources/aws_sqs_source_0.1.json

```diff
@@ -11,7 +11,8 @@
     "aws_queue_name_or_arn": {
       "title": "Queue Name",
       "description": "The SQS Queue Name or ARN",
-      "type": "string"
+      "type": "string",
+      "pattern": "^https:\/\/sqs\\.([a-z]+-[a-z]+-[0-9])\\.amazonaws\\.com\/[0-9]{12}\/([^\/]+)$"
```

Comment:
in previous patterns I saw `\/` to be escaped, here it is not escaped

Reply:
This is the regex `^https://sqs\\.([a-z]+-[a-z]+-[0-9])\\.amazonaws\\.com/[0-9]{12}/([^/]+)$` but escaped. Same for webhook: `(http|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])` -> `(http|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])`. 

I tested the schemas with https://www.jsonschemavalidator.net/ when I opened the PR and they looked ok. Could you double check?